client/devdocs/design/index.jsx

```diff
@@ -172,6 +174,7 @@ export default class DesignAssets extends Component {
 					<BackButton readmeFilePath="back-button" />
 					<Badge readmeFilePath="badge" />
 					<Banner readmeFilePath="banner" />
+					<Breadcrumb searchKeywords="navigation" readmeFilePath="breadcrumb" />
```

Comment:
Seems like the `searchKeywords` prop is unnecessary and can be removed.

Reply:
👍

Reply:
I see, that's fine to keep if it's intentional 👍